.github/workflows/ci.yaml

```diff
@@ -56,11 +61,16 @@ jobs:
               run: composer update
 
             - if: matrix.php-version == '8.0'
-              run: composer update --ignore-platform-reqs=php
+              run: composer update --ignore-platform-req=php
 
-            - name: "Install PHPUnit"
+            - if: matrix.php-version != '8.0' 
+              name: "Install PHPUnit"
               run: vendor/bin/simple-phpunit install
 
+            - if: matrix.php-version == '8.0'
+              name: "Install PHPUnit for PHP 8"
+              run: SYMFONY_PHPUNIT_VERSION=9.4 vendor/bin/simple-phpunit install
```

Comment:
I think you need to set it as a real env variable before, like:
![image](https://user-images.githubusercontent.com/995707/102985796-0cb65000-4510-11eb-9d78-4a7525d139cb.jpeg)

Sorry I just have a screenshot 

I am currently on a phone 📱

Reply:
It worked perfectly! You are a genius! Thanks Oskar.